go-controller/pkg/metrics/ovn.go

```diff
@@ -380,4 +382,33 @@ func RegisterOvnControllerMetrics() {
 	go ovnControllerConfigurationMetricsUpdater()
 	// ovn-controller coverage show metrics updater
 	go coverageShowMetricsUpdater(ovnController)
+	// Set IP family metric
+	ipFamilyMetricsPublisher()
+}
+
+func ipFamilyMetricsPublisher() {
+	var metricIPFamily = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Namespace: MetricOvnkubeNamespace,
+		Subsystem: MetricOvnkubeSubsystemMaster,
+		Name:      "stack",
```

Comment:
@gvbalaji maybe "ip_family" or "ip_families" not stack?

Reply:
Thanks Dan. I had it as ip_family to start with but it was changed with some unintended copy-paste.:) I reverted to ip_family.